recipes/pint/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
With setuptools, it is pretty much always ```--single-version-externally-manged --record=record.txt```, otherwise we get versioned (eggs?) directories in site-packages.